package.json

```diff
@@ -70,5 +71,11 @@
     "*.sol": "yarn lint:sol",
     "*.ts": "yarn lint:ts",
     "*.js": "prettier --write"
+  },
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.45.0",
+    "@typescript-eslint/parser": "^5.45.0",
+    "eslint": "^8.28.0",
+    "prettier-plugin-organize-imports": "^3.2.1"
```

Comment:
Aren't this dev dependencies?

Reply:
Yep, interestingly enough this was added by installation script of the tool, moved to devDependencies